lib/assets/gql-generator.js

```diff
@@ -125,9 +134,27 @@ function resolveVariableType (type, gqlSchema, stack = 0, stackLimit = 5) {
     });
     return fieldObj;
   }
-  return type.toString().includes('[') ?
-    [getDefaultForType(argType.toString())] :
-    getDefaultForType(argType.toString());
+
+  // If type is an array, get the default value for the type and return the desired depth nested array
+  if (type.toString().includes('[')) {
+    const getArrayDepth = (value) => {
+        return (value.match(/\[/g)).length;
```

Comment:
@Akshay2402 This should work for expected cases but not sure what's the standard for edge cases like below.

```
[[String]
[String
String]
[Stri[ng]
```